modules/tensor_mechanics/src/materials/FiniteStrainCrystalPlasticity.C

```diff
@@ -68,6 +80,9 @@ FiniteStrainCrystalPlasticity::FiniteStrainCrystalPlasticity(const std::string &
 
   _s0.resize(_nss);
 
+  if ( _num_slip_sys_props > 0 )
+    _slip_sys_props.resize( _nss * _num_slip_sys_props );
+
```

Comment:
Would a user ever want to run crystal plasticity with num_slip_sys_props = 0? It seems like we should add an "else" here that errors out.